### DISCUSSION TITLE
How to add 'distributor' user in admin management

### DISCUSSION THREAD

**Discussion by nadhirains** [saleor/saleor#6763] [Q&A]
Hi, all!
I need to add distributor admin level in Saleor-Dashboard, especially to filter orders that only the signed in distributor can see, and manage stock for its own warehouse. 
Maybe I enable to modify from the core, but if there is any upgrade in Saleor, will the modification break the upgrade?

Thanks

**Comment by karolkielecki** [comment]
@nadhirains Hey! I'm afraid that you will have to modify the code. Let me check with the team. I'll get back to you as soon as I can.
